bwresources.py

```diff
@@ -462,15 +347,17 @@ def _name_to_id(self, attribute, setting):
                 ids.append(self.categories.ids[parent]["children"][child])
             return ids
 
-        elif attribute in ["parentCategory", "xparentCategory"]:
+        elif attribute in ["parentCategory", "xparentCategory", "parentCategories", "categories"]:
+            #plural included for get_charts syntax
```

Comment:
What happens if somebody passes `categories` or `parentCategories` in, say, a `get_mentions` call?  My guess is that the parameter will be silently ignored, in which case we'll want to be emphatic about this in our documentation.